cmd/utils/flags.go

```diff
@@ -1102,7 +1102,10 @@ func SetNodeConfig(ctx *cli.Context, cfg *node.Config) {
 	setgRPC(ctx, cfg)
 	setNodeUserIdent(ctx, cfg)
 
-	cfg.DBType = ctx.GlobalString(DbTypeFlag.Name)
+	cfg.DBType = node.DefaultDBType()
+	if dbtype := database.DBType(ctx.GlobalString(DbTypeFlag.Name)); dbtype.IsValid() {
```

Comment:
How about `fatal` in invalid db type case?

Reply:
review applied Thx :)